src/analysis/multiSpecies/Microbiome_Modeling_Toolbox/useDiet.m

```diff
@@ -0,0 +1,36 @@
+function [modelOut] = useDiet(modelIn, dietConstraints)
```

Comment:
This seems to be the same function as PairwiseInteractionModelling/useDiet
If so, remove one of them otherwise rename one of them